max-george/starter-code/server.js

```diff
@@ -0,0 +1,217 @@
+'use strict';
+
+const pg = require('pg');
+const fs = require('fs');
+const express = require('express');
+const bodyParser = require('body-parser');
+const PORT = process.env.PORT || 3000;
+const app = express();
+const conString = 'postgres://localhost:5432/kilovolt'; // done: Don't forget to set your own conString
+const client = new pg.Client(conString);
+client.connect();
+client.on('error', function(error) {
+  console.error(error);
+});
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.static('./public'));
+
+app.get('/new', function(request, response) {
+  response.sendFile('new.html', {root: './public'});
+});
+
+app.get('/articles', function(request, response) {
+  // REVIEW: This query will join the data together from our tables and send it back to the client.
+  // DONE: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
+  client.query(`SELECT *
+                FROM articles
+                INNER authors
+                ON articles.author_id = authors.author_id;`)
+  .then(function(result) {
+    response.send(result.rows);
+  })
+  .catch(function(err) {
+    console.error(err)
+  });
+});
+
+app.post('/articles', function(request, response) {
+  client.query(
+    `INSERT INTO
+     authors (author, authorUrl)
+     VALUES ($1, $2)
+     ON CONFLICT (author) DO NOTHING;`, // DONE: Write a SQL query to insert a new author, ON CONFLICT DO NOTHING
```

Comment:
You need quotation marks around all the fields that are in camel-case or your queries will not work.